transports/transports.go

```diff
@@ -88,3 +89,12 @@ func ListNames() []string {
 	sort.Strings(names)
 	return names
 }
+
+// TransportFromImageName converts an URL-like name into a types.ImageTransport or nil when unavailable.
+func TransportFromImageName(imageName string) types.ImageTransport {
+	parts := strings.SplitN(imageName, ":", 2)
```

Comment:
Please add a
```go
// Keep this in sync with alltransports.ParseImageName!
```
comment at the top of the function body.  (And maybe a similar comment to the `ParseImageName` function.)

Reply:
Done.